tests/periph_timer/main.c

```diff
@@ -20,8 +20,10 @@
 
 #include <stdio.h>
 #include <stdint.h>
+#include <stdatomic.h>
 #include <stdlib.h>
 
+#include "macros/units.h"
```

Comment:
```suggestion
#include "board.h"

#include "macros/units.h"
```

This fixes the failing test on usb-kw41z (and probably on other boards)

Reply:
Thanks. I'll fix tonight. (Feel free to (force) push instead, of you want.)